src/Components/PresentationalComponents/Kebab/KebabItems.js

```diff
@@ -5,10 +5,14 @@ import { FormattedMessage } from 'react-intl';
 
 // Workaround: Using button instead of DropdownItem as it causes error on Dropdown open
 // in the current version of Patternfly
-export const kebabItemDownloadPDF = (loading, downloadReport, props = {}) => (
+export const kebabItemDownloadPDF = (downloadReport, { ...props } = {}) => (
```

Comment:
Is there a reason why is there `{ ...props } = {}` instead of `props = {}`? if props are undefined, this will crash.

Reply:
It seems I did something else and forgot to revert back to the previous state =)